content/intro-to-storybook/react/ja/data.md

```diff
@@ -18,135 +18,173 @@ commit: '5da7c68'
 以下のコマンドを実行し必要な依存関係を追加しましょう:
 
 ```bash
```

Comment:
@bhbs one item related to this section "## コンテナーコンポーネント". If you could update it to match the current version [here](https://storybook.js.org/tutorials/intro-to-storybook/react/en/data/) we'd appreciate it. The reasoning behind this is that that reference to "smart and dumb" components is a bit outdated and I've removed it from the tutorial to avoid misleading our readers.

Let me know once you've updated it and we'll go from there.

Reply:
@jonniebigodes 
Thanks! updated 👍
I've fixed `screen.md` too.

Reply:
@bhbs first and foremost sorry for not being clear, and for that, I truly apologize. What I meant to say is that this paragraph in the data section:
```md
`TaskList` コンポーネントは、今のところ、それ自体では外部とのやりとりをしないので「presentational (表示用)」([このブログ記事](https://medium.com/@dan_abramov/smart-and-dumb-components-7ca2f9a7c7d0)を参照) として書かれています。データを中に入れるためにはデータプロバイダに繋ぐ必要があります。
```

Is now outdated, If you check the English version, we're phrasing it as follows:

```md
Our TaskList component as currently written is “presentational” in that it doesn’t talk to anything external to its own implementation. We need to wire it to a data provider to get data into it.
```

I rephrased it a while ago to avoid any confusion as the article is not accurate anymore, nor should it be referenced as the author does not support the container/presentational pattern. 

Let me know if you have any questions and we'll go from there. On a side note, thanks for updating the images and wording in the other sections. Appreciate it 🙏